Optimization.Tests/DeflatedSharpeRatioFitnessTest.cs

```diff
@@ -0,0 +1,53 @@
+﻿using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Optimization.Tests
+{
+
+    [TestFixture]
+    public class DeflatedSharpeRatioFitnessTest
+    {
+
+
+        [Test]
+        public void CalculateExpectedMaximumTest()
+        {
+            var unit = new DeflatedSharpeRatioWrapper();
+            unit.Initialize(null);
+            var actual = unit.CalculateExpectedMaximum();
+
+            Assert.AreEqual(4.2894064662728244, actual, 0.0000001);
```

Comment:
We're expecting the expected maximum to return the value from the example formula: ~0.1132 as this is fed directly into the deflation. The code here is consistent (I think) with the python but not with the formula. One is mu+sigma*(...) wheres the formula has sqrt(1/2.25.)